mainnet/snapshot/backup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# script to sync data into s3:
+
+set -ex
+if [ -z "$BACKUP_DIR" ]; then
+	echo "'BACKUP_DIR'  var must be set"
+	exit 1
+fi
+DATA_DIR=$GOPATH/src/github.com/QuarkChain/goquarkchain/cmd/cluster/qkc-data/
+DATE=`date +%Y-%m-%d.%H:%M:%S`
+OUTPUT_FILE=$BACKUP_DIR/$DATE.tar.gz
+LATEST_FILE=$BACKUP_DIR/LATEST
+# 3 day's backup
+RETENTION=3
+cd $DATA_DIR
+cd ..
+./stop.sh
```

Comment:
stop.sh is not here

Reply:
why merge

Reply:
@mask-pp   I tried "kill -9" and "kill"  ,it showed "kill -9" that was good effect